### PR TITLE
Typos from rsync history

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

**Comment by Gelma** [comment]
@peternewman @luzpaz @larsoner 

Let's focus also on this!

**Comment by larsoner** [comment]
@peternewman feel free to merge if you're happy

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -9052,6 +9065,7 @@ heighest->highest
 heighteen->eighteen
 heigt->height
 heigth->height
+heir->hier
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Heir is a valid English word, remove. I don't think hier is a valid English word even!

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9407,6 +9421,7 @@ imbedded->embedded
 imbedding->embedding
 imcoming->incoming
 imcomming->incoming
+imcompatible->incompatible
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the ility version too.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -11985,6 +12002,7 @@ mostlky->mostly
 mosture->moisture
 mosty->mostly
 mot->not
+motation->notation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest rotation, motivation

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -12005,6 +12023,7 @@ moutns->mounts
 movebackwrd->movebackward
 moveble->movable
 movei->movie, disabled due to assembly code
+movied->moved
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest movie

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -12485,6 +12504,7 @@ ocassionaly->occasionally
 ocassioned->occasioned
 ocassions->occasions
 occaison->occasion
+occasinally->occasionally
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add occasinal->occasional

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -13809,6 +13831,7 @@ pounts->points
 poverful->powerful
 poweful->powerful
 powerfull->powerful
+powerppc->powerpc
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Specific but relevant, although I wonder if the correction should be capitalised.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -16564,6 +16589,7 @@ sintakts->syntax
 sintax->syntax
 Sionist->Zionist
 Sionists->Zionists
+sise->size
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest sisal?

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -16602,10 +16628,12 @@ slect->select
 slected->selected
 slecting->selecting
 slection->selection
+sleepp->sleep
 slient->silent
 slighlty->slightly
 slighly->slightly
 slighty->slightly
+sligntly->slightly
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add slignt->slight

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -17393,6 +17422,7 @@ sumamry->summary
 sumary->summary
 sumbitted->submitted
 sumed-up->summed-up
+summarise->summarize
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: BE to AE remove.

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -17393,6 +17422,7 @@ sumamry->summary
...
 summay->summary
 summerized->summarized
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the s variant

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -19422,6 +19457,7 @@ versionm->version
 versionms->versions
 versionned->versioned
 versionning->versioning
+versios->versions
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the singular

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -3315,6 +3317,7 @@ circularly->circular, circularly,
 circulaton->circulation
 circumferencial->circumferential
 circumsicion->circumcision
+circumstnces->circumstances
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -4223,12 +4227,14 @@ connectibity->connectivity
 connectino->connection
 connectinos->connections
 connectionas->connections
+connecton->connection
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -5083,12 +5089,14 @@ custome->custom, costume,
 customicable->customisable, customizable,
 customie->customize
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 15]
```diff
@@ -5083,12 +5089,14 @@ custome->custom, costume,
...
 customied->customized
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 16]
```diff
@@ -5083,12 +5089,14 @@ custome->custom, costume,
...
+customise->customize
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 10 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 11 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 17]
```diff
@@ -7966,6 +7976,7 @@ extepects->expects
 exteral->external
 extered->exerted
 extermist->extremist
+extern->zextern
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: You tell me

File: codespell_lib/data/dictionary.txt (hunk 12 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Can you add the singular @Gelma `circumstnce->circumstance`

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: What was the context you found it in? Officially it is, but possibly not in a code context: https://en.wikipedia.org/wiki/PowerPC

I'd be tempted to say yes, and people can manually lower case it if required by code.

File: codespell_lib/data/dictionary.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I meant in addition, either z or s are acceptable depending on your location.

File: codespell_lib/data/dictionary.txt (hunk 15 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I meant in addition, either z or s are acceptable depending on your location.

File: codespell_lib/data/dictionary.txt (hunk 12 above)
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: OK. I leave it so

File: codespell_lib/data/dictionary.txt (hunk 15 above)
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Ok, you tell me.

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: This is the way is written as flag for configure (autogen) and so on.

File: codespell_lib/data/dictionary.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: No, it needs to suggest both, given I suspect there are more AE than BE speakers, it should probably look like `customie->customize, customise,`

File: codespell_lib/data/dictionary.txt (hunk 15 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: As above.

File: codespell_lib/data/dictionary.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done, thanks

File: codespell_lib/data/dictionary.txt (hunk 15 above)
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done thanks

**Comment by luzpaz** [comment]
@Gelma would you please rebase ? Thanks!

File: codespell_lib/data/dictionary.txt [hunk 18]
```diff
@@ -14155,6 +14181,7 @@ presisting->persisting
 presistion->precision
 presists->persists
 presitgious->prestigious
+presmissions->permissions
```
Comment (c72578) [inline codespell_lib/data/dictionary.txt]: Is this intended?
`presmissions->permissions`
Or should it be:
`premissions->permissions`
Like `premission->permission`, which is already in the dictionary

**Comment by Gelma** [comment]
> @Gelma would you please rebase ? Thanks!

Now I push

File: codespell_lib/data/dictionary.txt (hunk 18 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: "presmissions->permissions" is one of the errors found in the rsync history

**Comment by Gelma** [comment]
> @Gelma would you please rebase ? Thanks!

I'm going to push

**Comment by Gelma** [comment]
> One missed change

I forced push to fix everything.

**Pull request merged**